tests/test_3/run.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+"""
+Pulling available repositories with an invalid configuration file should fail
+"""
+
+from nesttests import *
+import os
+
+assert nest(chroot="chroot", config="/non_existent/not_existing_either.toml").pull().returncode == 1
```

Comment:
```suggestion
assert nest(chroot="/tmp/chroot", config="/non_existent/not_existing_either.toml").pull().returncode == 1
```

(In case the test fails, so the content can be auto-removed by the shell script)